Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -81,6 +81,32 @@ public bool EnableControllers
             private set { enableControllers = value; }
         }
 
+        [SerializeField]
+        [Tooltip("Default Left Controller Model")]
+        private GameObject leftControllerModel;
+
+        /// <summary>
+        /// Enable and configure the controller rendering for the Mixed Reality Toolkit
+        /// </summary>
+        public GameObject LeftControllerModel
+        {
+            get { return leftControllerModel; }
+            private set { leftControllerModel = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Default Right Controller Model")]
+        private GameObject rightControllerModel;
+
+        /// <summary>
+        /// Enable and configure the controller rendering for the Mixed Reality Toolkit
```

Comment:
should the tooltip info and summary match?

Reply:
I'll fix those later, with so much change, I want to make it "work" (tm) first.

Reply:
The tooltip should be a subset of the summary tag, up to and including a subset of 100% :)

That said, I am fine with Simon doing a doc comment pass as a separate issue, if that works best.